app/javascript/mastodon/components/animated_number.js

```diff
@@ -51,7 +51,7 @@ export default class AnimatedNumber extends React.PureComponent {
     const { direction } = this.state;
 
     if (reduceMotion) {
-      return obfuscate ? obfuscatedCount(value) : <ShortNumber value={value} />;
+      return (obfuscate === true) ? obfuscatedCount(value) : <ShortNumber value={value} />;
```

Comment:
I am not sure what this attempts to fix.

Reply:
AFAICT the value of `obfuscate` is not initialized when this component is loaded, so the conditional logic in the original code doesn't actually do anything. Unless I'm missing upstream initialization the the conditional as originally coded will always evaluate to `false`. If that's the intent, then the conditional is entirely unnecessary.